mteb/models/b1ade_models.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from functools import partial
+
+from mteb.model_meta import ModelMeta, sentence_transformers_loader
+
+b1ade_training_data = {
+    # We are in teh process of submitting a paper outlining our process of creating b1ade using model merging and knowledge distillation.
+    # Similar to mixedbread models, we do not train on any data (except the MSMarco training split) of MTEB.
+    "MSMARCO": [],
+}
+
+b1ade_embed = ModelMeta(
+    name="b1ade-embed",
+    languages=["eng-Latn"], # in format eng-Latn
+    revision="3bdac13927fdc888b903db93b2ffdbd90b295a69",
+    open_weights=True,
+    release_date="2025-03-10",
+    n_parameters=335_000_000,
+    memory_usage_mb=1278,
+    embed_dim=1024,
+    license="mit",
+    max_tokens=4096,
+    reference="https://huggingface.co/w601sxs/b1ade-embed",
```

Comment:
The model card on HF is quite lacking. I would love to see that one updated before they merge this in to avoid cluttering the benchmark with too many undocumented models.

Reply:
Readme is updated, but not all the way https://huggingface.co/w601sxs/b1ade-embed

Reply:
Thanks!